### DISCUSSION TITLE
Setting up paths to additional simlibs

### DISCUSSION THREAD

**Discussion by Detzi88** [cocotb/cocotb#3885] [Q&A]
Hi,
i use ghdl/questa as a simulator and am wondering how one would properly set paths to third party libraies like vivados unisim.
For questa i manually add the following to the modelsim.ini under "[Library]":
UNISIM = simlibs/unisim
which works but i would prefer to set this in the cocotb makefile. 
Anyone knows how i can do that?

**Comment by F2g-git** [comment]
Have you ever found an alternative? I always see the hint to incorporate cocotb into the vivado simulaiton flow, but unfortunately I dont understand how I would use that for individual modules of my complete design.

 Your approach at least alowed me to get my simulation working again, so thanks a lot.

**Comment by Detzi88** [comment]
Can't quite remember, but i think the right way was to inlcude the modelsim.ini that is generated when you compile the simlibs. There migth even be a variable that can be used to include the modelsim.ini in the cocotb makefile but i'd had to look that up.
